features/ouders.feature

```diff
@@ -19,236 +19,234 @@ Rule: De ouder in categorie 2 heeft ouderAanduiding "1" en de ouder in categorie
-    | burgerservicenummer | 999996162                                    |
+    | burgerservicenummer | 000007894                                    |
     | fields              | ouders.ouderAanduiding,ouders.naam.voornamen |
-    Dan heeft de ouder met ouderAanduiding '1' de volgende gegevens
-    | naam            | waarde |
-    | ouderAanduiding | 1      |
-    En heeft de ouder met ouderAanduiding '1' de volgende 'naam' gegevens
-    | naam      | waarde  |
-    | voornamen | Chantal |
-    En heeft de ouder met ouderAanduiding '2' de volgende gegevens
+    Dan heeft de response een persoon met een 'ouder' met de volgende gegevens
+    | naam            | waarde  |
+	| naam.voornamen  | Chantal |
+    | ouderAanduiding | 1       |
+    En heeft de persoon een 'ouder' met de volgende gegevens
     | naam            | waarde |
+	| naam.voornamen  | Wieger |
     | ouderAanduiding | 2      |
-    En heeft de ouder met ouderAanduiding '2' de volgende 'naam' gegevens
-    | naam      | waarde    |
-    | voornamen | Wieger    |
 
 @gba
 Rule: de geleverde oudergegevens zijn de gegevens zoals die staan op de persoonslijst van de gevraagde persoon
   # Bij het raadplegen van een persoon worden alleen gegevens uit de persoonslijst van de gevraagde persoon gebruikt, en nooit gegevens van de persoonslijst van de ouder
 
   @gba
   Scenario: De ouder heeft geslachtswijziging ondergaan, maar de gevraagde persoon erkent dit niet
-    Gegeven de persoon met burgerservicenummer 555550003 heeft de volgende ouders in de registratie
-    | Categorie | Burgerservicenummer (01.20) | Voornamen | Geslachtsaanduiding (04.10) |
-    | 2         | 555550004                   | Karel     | M                           |
-    | 3         | 555550005                   | Ellen     | V                           |
-    En de ouder met burgerservicenummer 555550004 heeft haar geslacht en voornamen gewijzigd zodat de volgende persoonsgegevens in de registratie op haar persoonslijst staan:
-    | Categorie | Voornamen | Geslachtsaanduiding (04.10) |
-    | 1         | Charlotte | V                           |
-    | 51        | Karel     | M                           |
+    Gegeven de persoon met burgerservicenummer '000007895' heeft een ouder '1' met de volgende gegevens
+    | burgerservicenummer (01.20) | voornamen (02.10) | geslachtsaanduiding (04.10) |
+    | 000007896                   | Ellen             | V                           |
+    En de persoon heeft een ouder '2' met de volgende gegevens
+    | burgerservicenummer (01.20) | voornamen (02.10) | geslachtsaanduiding (04.10) |
+    | 000007897                   | Karel             | M                           |
+    En de ouder '2' is gecorrigeerd naar de volgende gegevens
+    | burgerservicenummer (01.20) | voornamen (02.10) | geslachtsaanduiding (04.10) |
+    | 000007897                   | Charlotte         | V                           |
```

Comment:
hiermee is de feature precies niet af. wat ontbreekt is invulling aan "maar de gevraagde persoon erkent dit niet"

dit betekent m.i. dat je:
1. in Gegeven ook de ouder als persoon moet opnemen (Gegeven de persoon met burgerservicenummer '000007897' heeft de volgende gegevens)
2. in Gegeven voor de persoon ouder 2 moet worden teruggecorrigeerd naar Karel

dan heb je de situatie dat persoon 000007897 als voornaam "Charlotte" heeft, maar ouder 2 met bsn 000007897 nog/weer als voornaam "Karel" heeft.

Het doel is te tonen dat de API dan de voornaam van ouder 2 levert en niet de gerelateerde persoon. Dus de API levert dan "Karel" als voornaam.

Reply:
Hierover had ik bovenin deze PR al een vraag gesteld. Ik begrijp nu hoe dit in de datebase tot uitdrukking komt.
Ik denk niet dat we het zo moeilijk hoeven te maken als je het hier beschrijft, we kunnen n.m.m. volstaan met het aan dit scenario toevoegen van nog een 'En de ouder '2' is gecorrigeerd naar de volgende gegevens' waarin de naam en geslacht weer worden teruggewijzigd.